### DISCUSSION TITLE
Date & Time - Fields

### DISCUSSION THREAD

**Discussion by esoleyman** [Syriac-Localization/syriac-localization.github.io#1] [General]
future-one, in {0} [day / month / year] -- ܒ{0} ܝܘܡܐ ܇ ܝܪܚܐ ܇ ܫܢܬܐ 

If prompt is **future-one**, we should just use the number one followed by the [field]; e.g. ܚܕܐ ܫܢܬܐ، ܚܕ ܝܪܚܐ، ܚܕ ܝܘܡܐ

**Comment by SargisYonan** [comment]
<img width="435" alt="Screen Shot 2023-06-22 at 10 52 29 PM" src="https://github.com/Syriac-Localization/syriac-localization.github.io/assets/5349027/085f8368-b74a-4d49-8276-c22a7480fdc3">
How would we dynamically write out the number in this case? It looks like we can get any integer value in this case

**Reply by esoleyman** [reply]
There's a marked difference between **future-one** and **future-other**

future-one: ܒܚܕܐ ܫܢܬܐ
future-other: ܒ{0} ܫܢܝ̈ܐ

We change the formatting for future-one only and leave future-other as-is for most of them

**Reply by esoleyman** [reply]
Arabic does the same thing for any **future-one**. It gets rid of the dynamic number and just writes it out.

future-one | in {0} year | ✓ | خلال سنة واحدة |  
future-one | in {0} quarter | ✓ | خلال ربع سنة واحد |  
future-one | in {0} month | ✓ | خلال شهر واحد |
